dist/redux-saga-monitor.js

```diff
@@ -0,0 +1,1807 @@
+	  };
+	}
+
+	function argToString(arg) {
+	  return typeof arg === 'function' ? '' + arg.name : typeof arg === 'string' ? '\'' + arg + '\'' : arg;
+	}
+
+	function logResult(_ref, formatter, ignoreResult) {
+	  var status = _ref.status;
+	  var result = _ref.result;
+	  var error = _ref.error;
+	  var duration = _ref.duration;
+
+
+	  if (status === RESOLVED && !ignoreResult) {
+	    if (_utils.is.array(result)) {
+	      formatter.addValue(' → ');
+	      formatter.addValue(result);
+	    } else formatter.appendData('→', result);
+	  } else if (status === REJECTED) {
+	    formatter.appendData('→ ⚠', error);
+	  } else if (status === PENDING) formatter.appendData('⌛');else if (status === CANCELLED) formatter.appendData('→ Cancelled!');
+
+	  if (status !== PENDING) formatter.appendData('(' + duration.toFixed(2) + 'ms)');
+	}
+
+	function logFormatter() {
+	  var logs = [];
+	  var suffix = [];
+
+	  function add(msg) {
+	    for (var _len2 = arguments.length, args = Array(_len2 > 1 ? _len2 - 1 : 0), _key2 = 1; _key2 < _len2; _key2++) {
+	      args[_key2 - 1] = arguments[_key2];
+	    }
+
+	    // Remove the `%c` CSS styling that is not supported by the Node console.
+	    if (!IS_BROWSER && typeof msg === 'string') {
+	      var prevMsg = msg;
+	      msg = msg.replace(/^%c\s*/, '');
+	      if (msg !== prevMsg) {
+	        // Remove the first argument which is the CSS style string.
+	        args.shift();
+	      }
+	    }
+	    logs.push({ msg: msg, args: args });
+	  }
+
+	  function appendData() {
+	    for (var _len3 = arguments.length, data = Array(_len3), _key3 = 0; _key3 < _len3; _key3++) {
+	      data[_key3] = arguments[_key3];
+	    }
+
+	    suffix = suffix.concat(data);
+	  }
+
+	  function addValue(value) {
+	    if ((0, _isPrimitive2.default)(value)) add(value);else {
+	      // The browser console supports `%O`, the Node console does not.
+	      if (IS_BROWSER) add('%O', value);else add('%s', (0, _util.inspect)(value));
+	    }
+	  }
+
+	  function addCall(name, args) {
+	    if (!args.length) add(name + '()');else {
+	      add(name);
+	      add('(');
+	      args.forEach(function (arg, i) {
+	        addValue(argToString(arg));
+	        addValue(i === args.length - 1 ? ')' : ', ');
+	      });
+	    }
+	  }
+
+	  function getLog() {
+	    var msgs = [];
+	    var msgsArgs = [];
+	    for (var i = 0; i < logs.length; i++) {
+	      msgs.push(logs[i].msg);
+	      msgsArgs = msgsArgs.concat(logs[i].args);
+	    }
+	    return [msgs.join('')].concat(msgsArgs).concat(suffix);
+	  }
+
+	  return {
+	    add: add, addValue: addValue, addCall: addCall, appendData: appendData, getLog: getLog
+	  };
+	}
+
+	var logSaga = function logSaga() {
+	  console.log('');
+	  console.log('Saga monitor:', Date.now(), new Date().toISOString());
+	  logEffectTree(0);
+	  console.log('');
+	};
+
+	// Export the snapshot-logging function to run from the browser console or extensions.
+	if (IS_BROWSER) {
+	  window.$$LogSagas = logSaga;
```

Comment:
Please choose a more unique global name, such as `__REDUX_SAGA_MONITOR_LOG_SAGAS__ `. Please see the reasoning here: https://github.com/zalmoxisus/redux-devtools-extension/issues/202

If you're in doubt that the long name would be difficult to run from the browser developer tools console, the console has got a decent auto-complete and command history, so you will only need to enter (read: copy-paste) that once.